src/main/java/org/broadinstitute/hellbender/tools/spark/sv/CollectISizeDistributionSpark.java

```diff
@@ -0,0 +1,175 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import htsjdk.samtools.SAMFileHeader;
+import htsjdk.samtools.metrics.MetricsFile;
+import htsjdk.samtools.util.IOUtil;
+import org.apache.spark.api.java.JavaRDD;
+import org.apache.spark.api.java.JavaSparkContext;
+import org.broadinstitute.hellbender.cmdline.Argument;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.SparkProgramGroup;
+import org.broadinstitute.hellbender.engine.filters.AlignmentAgreesWithHeaderReadFilter;
+import org.broadinstitute.hellbender.engine.filters.ReadFilter;
+import org.broadinstitute.hellbender.engine.spark.GATKSparkTool;
+import org.broadinstitute.hellbender.metrics.MetricAccumulationLevel;
+import org.broadinstitute.hellbender.metrics.MetricsUtils;
+import org.broadinstitute.hellbender.tools.picard.analysis.InsertSizeMetrics;
+import org.broadinstitute.hellbender.utils.R.RScriptExecutor;
+import org.broadinstitute.hellbender.utils.io.Resource;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+import java.io.File;
+import java.util.*;
+
+@CommandLineProgramProperties(
+        summary        = "Program to collect insert size distribution information in SAM/BAM file(s)",
+        oneLineSummary = "CollectInsertSizeDistr on Spark",
+        programGroup = SparkProgramGroup.class)
+public final class CollectISizeDistributionSpark extends GATKSparkTool {
+    private static final long serialVersionUID = 1L;
+
+    @Argument(doc = "Should an output plot be created?")
+    public boolean PRODUCE_FILES = false;
+
+    @Argument(doc = "A local path to file to write insert size metrics to.",
+            shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME, fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME,
+            optional = true)
+    public String OUTPUT;
+
+    @Argument(doc="PDF File to write histogram chart to.",
+            shortName="H")
+    public File HISTOGRAM_FILE;
+
+    @Argument(doc="Similar to Picard CollectInsertSizeMetrics::DEVIATIONS.")
```

Comment:
`Similar to Picard CollectInsertSizeMetrics::DEVIATIONS.` is  not useful documentation - describe here what this argument does